README.md

```diff
@@ -5,757 +5,886 @@
+With Microstates added to your project, you get:
 
-* _Typed_ means that microstates know about the kind of data it contains and transitions that can be
-  performed on that data.
-* _Composable_ means that simple types can be grouped to describe complex data structures.
-  Microstates knows how to transition complex data structures in an immutable way.
-* _Immutable_ means that state in the microstate cannot be modified. To change the state, you must invoke a transition.
-* _Transition_ is an operation that has access to the current state, might perform computations/transformations on it and returns a new state.
+* 🍇 Composable type system
+* 🍱 Reusable state atoms
+* 💎 Pure immutable state transitions without writing reducers
+* ⚡️ Lazy and synchronous out of the box
+* 🦋 Most elegant way to express state machines
+* 🎯 Transpilation free type system
+* 🔭 Optional integration with Observables
+* ⚛ Use in Node.js and browser
 
-To add microstates to your project,
+But, most imporantly, Microstates makes working with state fun.
 
-```bash
-npm install --save microstates
+**When was the last time you had fun working with state?**
 
-# yarn add microstates
-```
+For many, the answer is probably never, because state management in JavaScript is an endless game of compromises. You can choose to go fully immutable and write endless reducers. You can go mutable and everything becomes an observable. Or you can `setState` and lose the benefits of serialization and time travel debugging.
 
-Let's see some code,
+Unlike the view layer, where most frameworks agree on some variation of React's concept of components, state management is a lot less certain. This is because none of the available tools strike the balance that React components introduced to the view layer.
 
-```js
-import { create } from 'microstates';
+React components have a tiny API. They are functional, simple and extremely reusable. The tiny API gives you high productivity for little necessary knowledge. Functional components are predictable and easy to reason about. They are conceptually simple, but simplicity hides an underlying architecture geared for performance. Their simplicity, predictability and isolation makes them composable and reusable.
 
-create(Number, 42).increment().state;
-//=> 43
+These factors combined are what make React style components easy to work with and ultimately fun to write. A Tiny API abstracting a sophisticated architecture that delivers performance and is equaly useful on small and big projects is the outcome that we set out to achieve for state management with Microstates.
 
-class MyCounter {
-  count = Number;
-}
+It's not easy to find the right balance between simplicity and power, but considering the importance of state management in web applications, we believe it's a worthy challenge.
 
-// creating microstate from composed types
-create(MyCounter).state;
-// => { count: 0 }
+# Why Microstates?
```

Comment:
When I see this. I'm like "Why Microstates? Is that what I'm reading right now?" I think this whole section should be in a blog post.